lerobot/scripts/control_robot.py

```diff
@@ -515,6 +517,11 @@ def replay(
         nargs="*",
         help="Any key=value arguments to override config values (use dots for.nested=overrides)",
     )
+    parser_record.add_argument(
```

Comment:
I went back to using a bool here since it's meant to be a flag for assigning rewards or not.
Lmk if you think it still should be an int.

Reply:
okk